### PR TITLE
docs: Update documentation to include the data browser

### PR DESCRIPTION
**Pull request by flavens**
resolves DEV-1667

### CODE REVIEW

**Comment by flavens** [comment]
@mdelez @Rebeccasigloch FYI I did not replace all screenshots everywhere because I don't have the same project as André used. I mostly used Incunabula or BEOL. I had no time yet to install and test the mkdocs video plugin. Next time I hope. You can use the preview functionality in VSCode to see how the markdowns are rendered (however the size of some images will be smaller in docs.dasch.swiss).

**Review by mdelez — APPROVED** [review]
thanks :)

**Comment by flavens** [comment]
> thanks :)

what? not even a typo somewhere?! that's good :)

File: docs/user-guide/project.md [hunk 1]
```diff
@@ -2,52 +2,77 @@
...
 
-![Project list and project menu](../assets/images/dashboard.png)*<https://admin.dasch.swiss/dashboard> - By clicking on the project name, you get access to the full project information.*
+Active projects are tagged with the status "Active". Projects tagged as "Deactivated" can no longer be edited. Only a system admin is allowed to change the project status.
 
-A system administrator can create a new research project. This currently requires essential information such as the project name, a shortcode and a shortname (both provided by DaSCH). A short project description is optional, but highly recommended.
+ The setting icon on the top right of each project tile is displayed only for projects and system admins. It leads to the project settings page.
+
+![Project list and project menu](../assets/images/dashboard.png)*<https://admin.dasch.swiss/dashboard> - DSP-APP projects overview page.*
+
+ By clicking on the button `Browse Data`, you go to the **project workspace**. You get access to the project information, the data model (if you are logged in), and the project settings (if you are logged in and a project admin or a system admin).
```
Comment (Rebeccasigloch) [inline docs/user-guide/project.md:13]: (if you are logged in as a ....)

File: docs/user-guide/project.md [hunk 2]
```diff
@@ -2,52 +2,77 @@
...
+
+![Collaboration page](../assets/images/project-settings.png)*<https://admin.dasch.swiss/project/0803/collaboration> - Project members page where project admin and system admin can add new users to the team.*
 
-![Collaboration page](../assets/images/project-members.png)*<https://admin.dasch.swiss/project/0803/collaboration> - Project members page where project admin and system admin can add new user to the team.*
+From the project workspace, as a project member, you can:
 
-<!-- >> **_NOTE:_** **Permissions for project admins to add new users as project member will be implemented soon.** -->
+* View the project information
+* View the data model
+* Get the list of resource classes with the number of entries, and click on the tiles to make a search by resource class.
```
Comment (Rebeccasigloch) [inline docs/user-guide/project.md:48]: I would add a new bullet point *Create or edit resources

**Review by Rebeccasigloch — APPROVED** [review]
Looks good to me thanks

**Pull request merged**